compiler/rustc_borrowck/src/region_infer/opaque_types.rs

```diff
@@ -96,7 +96,7 @@ impl<'tcx> RegionInferenceContext<'tcx> {
                         .iter()
                         .find(|ur_vid| self.eval_equal(vid, **ur_vid))
                         .and_then(|ur_vid| self.definitions[*ur_vid].external_name)
-                        .unwrap_or(infcx.tcx.lifetimes.re_root_empty),
+                        .unwrap_or(infcx.tcx.lifetimes.re_erased),
```

Comment:
This is problematic (leads to an ICE when compiling rustc_data_structures). I have to look into this code a bit and figure out a fix.